deep_qa/training/multi_gpu.py

```diff
@@ -0,0 +1,57 @@
+from keras.layers import merge
+from keras.layers.core import Lambda
+from .models import DeepQaModel
+import tensorflow as tf
+
+
+def make_parallel(model: DeepQaModel, gpu_count: int) -> DeepQaModel:
+
+    """
+    :param model: An instance of a DeepQaModel.
```

Comment:
We're using a different docstring format.  This won't work with our doc tools (and even if I'm wrong and it does, we should still just standardize on a single docstring format, for consistency and readability across the library).